spec/ttml1.xml

```diff
@@ -8106,7 +8123,7 @@ based timing or event based timing in <bibref ref="smil21"/>, where the specific
   &lt;div&gt;
     &lt;p timeContainer=&quot;seq&quot;&gt;
       Hello
-      &lt;span&gt;Bonjour&lt;/span&gt;
+      &lt;p&gt;Bonjour&lt;/p&gt;
```

Comment:
This is not valid TTML - a `p` cannot contain a `p`.

Reply:
@nigelmegitt Thanks for the catch!